calculator33.py

```diff
@@ -1,26 +1,10 @@
-operations_count = 0
+def integer_overflow_noncompliant():
+    # Noncompliant: Number larger than limit of the datatype is stored.
+    arr = np.array([[100000000]], dtype=np.int8)
 
-def main():
-    ask_again = True
-    while(ask_again):
-        a = input("Enter the numerator: ")
-        b = input("Enter the denominator: ")
-        result = perform_division(a,b)
-        print(result)
-        ask_again = input("Do you want to perform another operation? Enter yes or no: ")
-        if(ask_again == 'yes'):
-            ask_again = True
-        else:
-            ask_again = False
-            print("You performed " + str(operations_count) + " operations, bye!")
 
-
-def perform_division(a,b):
-    global operations_count
-    try:
-        operations_count += 1
-        return int(a)/int(b)
-    except Exception as e:
-        pass
-
-main()
\ No newline at end of file
+def create_session_noncompliant():
+    import boto3
+    # Noncompliant: uses hardcoded secret access key.
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded AWS Secret Access Key ID. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources that use this access key and using IAM roles instead of hardcoded access keys for IAM users. If you must use IAM users, secure their secrets outside your code.

[Learn more about the use of hardcoded credentials](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html)